br/pkg/stream/rewrite_meta_rawkv_test.go

```diff
@@ -468,18 +468,14 @@ func TestRewriteTableInfoForExchangePartition(t *testing.T) {
 		Name:      model.NewCIStr(tableName1),
 		Partition: &pi,
 	}
-	db1 := model.DBInfo{
-		ID: dbID1,
-	}
+	db1 := model.DBInfo{}
 
 	// construct table t2 without partition.
 	t2 := model.TableInfo{
 		ID:   tableID2,
 		Name: model.NewCIStr(tableName2),
 	}
-	db2 := model.DBInfo{
-		ID: dbID2,
```

Comment:
Why remove this ID？

Reply:
Because it's not used in this test :thinking: .

Reply:
Well, that test passes. I guess this will probably be fine.